src/PerfView64/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Reflection;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("PerfView64")]
+[assembly: AssemblyDescription("A 64-bit launcher for PerfView")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Microsoft")]
+[assembly: AssemblyProduct("PerfView")]
+[assembly: AssemblyCopyright("Copyright © Microsoft 2010")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+// Setting ComVisible to false makes the types in this assembly not visible
+// to COM components.  If you need to access a type in this assembly from
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+//In order to begin building localizable applications, set
+//<UICulture>CultureYouAreCodingWith</UICulture> in your .csproj file
+//inside a <PropertyGroup>.  For example, if you are using US english
+//in your source files, set the <UICulture> to en-US.  Then uncomment
+//the NeutralResourceLanguage attribute below.  Update the "en-US" in
+//the line below to match the UICulture setting in the project file.
+
+//[assembly: NeutralResourcesLanguage("en-US", UltimateResourceFallbackLocation.Satellite)]
+
+
+//[assembly: ThemeInfo(
+//    ResourceDictionaryLocation.None, //where theme specific resource dictionaries are located
+//                                     //(used if a resource is not found in the page,
+//                                     // or application resource dictionaries)
+//    ResourceDictionaryLocation.SourceAssembly //where the generic resource dictionary is located
+//                                              //(used if a resource is not found in the page,
+//                                              // app, or any theme specific resource dictionaries)
+//)]
+
+
+// Version information for an assembly consists of the following four values:
+//
+//      Major Version
+//      Minor Version
+//      Build Number
+//      Revision
+//
+// You can specify all the values or you can default the Build and Revision Numbers
+// by using the '*' as shown below:
+// [assembly: AssemblyVersion("1.0.*")]
+[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyFileVersion("1.9.53")]
```

Comment:
Don't make this version the same as the PerfView version, as I don't want to have to keep it in sync.   It is a wrapper and its version is 1.0.0, and it will probably never change again.